include/communication_interface.h

```diff
@@ -80,6 +80,9 @@ class CommunicationInterface {
                            std::string driver_status_string = "");
 
   void PublishDriverStatus(bool success, std::string driver_status_string = "");
+  void PublishBoolToChannel(int64_t utime, std::string lcm_channel, bool data);
```

Comment:
Can you also replace the other instances of bool message publishing with calls to this function?